src/main/java/org/cbioportal/web/util/ClinicalDataBinUtil.java

```diff
@@ -310,17 +310,33 @@ public List<ClinicalDataBin> calculateStaticDataBins(
         List<String> filteredUniqueSampleKeys,
         List<String> filteredUniquePatientKeys
     ) {
-        return NewClinicalDataBinUtil.calculateStaticDataBins(
```

Comment:
This is to revert the legacy implementation for the legacy endpoint. We can't use `NewClinicalDataBinUtil.calculateDynamicDataBins` for legacy endpoint anymore since the underlying logic changed significantly.